src/OrchardCore.Modules/OrchardCore.Security/Services/PermissionsHeaderPolicyProvider.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using Microsoft.AspNetCore.Http;
+
+namespace OrchardCore.Security.Services
+{
+    public class PermissionsHeaderPolicyProvider : HeaderPolicyProvider
+    {
+        public override void ApplyPolicy(HttpContext httpContext)
+        {
+            httpContext.Response.Headers[SecurityHeaderNames.PermissionsPolicy] = String.Join(SecurityHeaderDefaults.PoliciesSeparater, Options.PermissionsPolicy);
```

Comment:
apparently the PoliciesSeparater is not just used to split. Just use an inlined string here.

Reply:
Separator could be used for both split and join